tests/conftest.py

```diff
@@ -58,15 +58,15 @@
     "F a",
     "F(a & b)",
     "F(a | b)",
-    # "F(a & last)",
+    "F(a & last)",
     "G a",
     "F(G(a))",
     "G(F(a))",
     "G(true)",
     "F(true)",
     "G(false)",
     "F(false)",
-    # "last",
+    "last",
     # "end",
```

Comment:
do we support LTLfEnd? If yes, we could uncomment this

Reply:
Actually, we don't because I disallowed it inside the parser.
The class LTLfEnd seems to work as expected (for example it's still used to compute the delta function of Next and WeakNext).
However, as we discussed, there is no way to reach End by means of Next and WeakNext (we're not in a true empty trace version of LTLf, yet).
I could enable it, so we can state `"End"` to request an empty trace, but `"X End"` would not be satisfied by traces of lenght 1.

Reply:
oh ok, nevermind. Happy to merge.